keanu-project/src/main/java/io/improbable/keanu/vertices/bool/probabilistic/Flip.java

```diff
@@ -8,41 +8,42 @@
 
 public class Flip extends ProbabilisticBool {
 
-    private final Vertex<Double> probTrue;
+    private final Vertex<Double> probabilityTrue;
```

Comment:
Feels a bit odd to have `logProb` and now `probabilityTrue`. Let's pick `prob` or `probability` throughout?